frontend/src/metabase/components/ErrorPages/use-error-info.ts

```diff
@@ -9,6 +9,18 @@ import { UtilApi, MetabaseApi, SessionApi } from "metabase/services";
 import type { ErrorPayload, ReportableEntityName } from "./types";
 import { getEntityDetails, hasQueryData } from "./utils";
 
+const maybeSerializeError = (key: string, value: any) => {
+  if (value?.constructor.name === "Error") {
```

Comment:
I _think_ it might be preferable to do `value instanceof Error` to capture custom err values that have extended the Error type:
```ts
class CustomError extends Error {};
const err1 = new Error('err1');
const err2 = new CustomError('err2');
err1?.constructor.name === "Error" // true
err1 instanceof Error // true
err2?.constructor.name === "Error" // false
err2 instanceof Error // true
```